manifests/m/Microsoft/WindowsTerminal/Preview/1.21.1272.0/Microsoft.WindowsTerminal.Preview.installer.yaml

```
@@ -0,0 +1,31 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Microsoft.WindowsTerminal.Preview
+PackageVersion: 1.21.1272.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.19041.0
+InstallerType: msix
+InstallModes:
+- silent
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.UI.Xaml.2.8
+    MinimumVersion: 2.8.4

```

```suggestion
  PackageDependencies:
  - PackageIdentifier: Microsoft.UI.Xaml.2.8
    MinimumVersion: 8.2305.5001.0
```

cc @denelon / @DHowett to apply the changes please. This is to point to the correct package version in the repo: https://github.com/microsoft/winget-pkgs/tree/master/manifests/m/Microsoft/UI/Xaml/2/8/8.2305.5001.0